### ISSUE TITLE
Many examples of copulas incorrectly tagged as AUX, or auxiliaries incorrectly labeled as v.stative

### ISSUE THREAD

**Issue by nschneid** [dimsum16/dimsum-data#2]
```
$ fgrep 'AUX' dimsum16.train | fgrep 'v.stative' | wc -l
     342
$ fgrep 'AUX' dimsum16.train | fgrep -v 'v.stative' | wc -l
    2352
$ fgrep 'v.stative' dimsum16.train | fgrep -v 'AUX' | wc -l
    3089
```

E.g., the following is a copula, so should be `VERB` rather than `AUX`:

```
1       Is      is      AUX     O                       v.stative       lowlands-4
2       your    your    DET     O                               lowlands-4
3       cat     cat     NOUN    O                       n.animal        lowlands-4
4       safe    safe    ADJ     O                               lowlands-4
5       from    from    ADP     O                               lowlands-4
6       disease disease NOUN    O                       n.state lowlands-4
```

Whereas the following is an auxiliary, so there should be no supersense:

```
1       Were    were    AUX     O                       v.stative       lowlands-32
2       ordered ordered VERB    O                       v.communication lowlands-32
3       to      to      ADP     O                               lowlands-32
4       chant   chant   VERB    O                       v.communication lowlands-32
5       praise  praise  NOUN    O                       n.communication lowlands-32
```

**Comment by nschneid** [comment]
I see two options for addressing this: (a) Someone at CPH goes through and fixes the 342 instances, or (b) I have my annotator label auxiliaries with <code>`a</code> instead of a supersense, after which the POS tags can be updated accordingly. The downside of option (b) is more tokens/an additional distinction for him to have to worry about.

**Comment by dirkhovy** [comment]
I went through both Lowlands and Ritter and fixed all AUX
- if they were copula and had a supersense, I changed them to be VERB
- if they were true auxiliaries, I removed the supersense

I pushed the changed files in the conversion/original folder, but couldn't run the convert.sh script, which gave me an AssertionError. 

@andersjo, do you know what went wrong there? Could you fix that and run convert.sh?

Cheers,

Dirk

**Comment by dirkhovy** [comment]
Never mind, found the probem: trailing "I-verb." supersense. Manually changed it in the Ritter data and reran convert.sh successfully (changed the assert statement there slightly to output the offending line and token).

Should we announce another release, or do you want to take a look first, @nschneid?

Cheers,

Dirk

**Comment by nschneid** [comment]
Thanks! Let me take a look.

**Comment by nschneid** [comment]
A few remain in the Reviews data, which I can fix.

**Comment by nschneid** [comment]
Another thing: In Ritter, URLs are POS-tagged as `X`; in Lowlands they are `PROPN`.

**Comment by nschneid** [comment]
I reviewed the 16 Reviews cases:
- 8 were apparent (or arguable) treebanking errors, especially cases where a predicate adjective was mistagged as a verb. E.g.: "...replace the board that **is** cracked (the crack is deep enough..."
- 3 were cases where the aux headed an MWE, which was labeled `stative` as a whole. E.g.: "My wife and kids **ca**_n't_get_enough ."
- 5 supersense labeling errors. Changed `stative` → <code>`a</code> in the annotation tool, though it's probably not worth re-releasing STREUSLE over these. So let's leave them alone in the DiMSUM release for now.
